src/Transfer/Adapter/FilterPluginManager.php

```diff
@@ -20,16 +21,19 @@
  */
 class FilterPluginManager extends BaseManager
 {
-    /**
-     * Default set of filters
-     *
-     * @var array
-     */
-    protected $aliases = [
```

Comment:
@Ocramius yes and no.

Prior to zend-filter 2.6.0, `Zend\Filter\FilterPluginManager` did not define any `$aliases`, which made adding them to this extension safe. However, part of the 2.6.0 update was separating invokables into aliases + factories. That change has made it so that adding the aliases property in this extension breaks expectations, as other filters expected to be present are missing. The approach being done in this patch is to merge this "default set" with those defined in the parent, via the constructor, which is both backwards and forwards compatible.